CONTRIBUTING.md

```diff
@@ -287,6 +287,66 @@ If the SVG is sourced from:
 
 In general, make sure the URL does not contain any tracking identifiers.
 
+#### Aliases
+
+Lastly, we aim to provide aliases of three types for various reasons. Each type of alias and their motivation can be found below. If you're unsure, you can mention an alias you're considering in your Pull Request so it can be discussed.
+
+##### Also Known As
+
+We collect "also known as" names to make it easier to find brands that are known by different names or by their abbreviation/full name. To add an "also known as" name you add the following to the icon data:
+
+```json
+{
+  "title": "the original title",
+  "aliases": {
+    "aka": [
+      "tot"
+    ]
+  }
+}
+```
+
+Where the string is **different** from the original title as well as all other strings in the list.
+
+##### Duplicates
+
+We collect the name of duplicates, brands that use the same logo but have a different name, to prevent duplicating an SVG while at the same time making the SVG available under the name of the duplicate. To add a duplicate you add the following to the icon data:
+
+```json5
```

Comment:
Minor tyo:
```suggestion
```json
```

Reply:
Note that the code block has comments, it is in JSON5 format.

Reply:
After a bit of searching, I think that [`jsonc` (JSON with comments)](https://stackoverflow.com/a/55708553/9167585) fits better the needs here. Note that is supported by ]Github Flavored Markdown code blocks (linguist)](https://github.com/github/linguist/blob/e6860ab64b463e022294b0f71d468d5243687027/lib/linguist/languages.yml#L2569).

Reply:
Indeed, alternative could be to use `js` instead of `json5` but I would say that is even more confusing.